README.md

```diff
@@ -675,23 +675,26 @@ Set-point types: normal, boost, vacation.
 
 #### Interfaces
 
-Type | Interface               | Value type | Properties           | Description
------|-------------------------|------------|----------------------|------------------
-in   | cmd.lock.get_report     | null       |                      |
-in   | cmd.lock.set            | bool       |                      | Use true to secure a lock and false to unsecure
-in   | cmd.lock.set_with_code  | str_map    |                      | Used to lock/unlock locks required PIN/RFID, {“op”:”lock”, ”code_type”:”pin”, ”12345” }
-out  | evt.lock.report         | bool_map   | timeout_s, lock_type | value = {"is_secured":true, "door_is_closed":true, "bolt_is_locked":true, "latch_is_closed":true}
--|||
-in   | cmd.open.get_report     | null       |                      | 
-out  | evt.open.report         | bool       | true = open          | Used to report if the door is open or closed
--|||
-in   | cmd.auto_lock.set        | bool      | true = auto-lock enabled | Enable/disable auto-lock feature
-in   | cmd.auto_lock.get_report | null      |                          | 
-out  | evt.auto_lock.report     | bool      | true = auto-lock enabled |
--|||
-in   | cmd.volume.set           | int       | `min_volume`-`max_volume`
-in   | cmd.volume.get_report    | null      |
-out  | evt.volume.report        | int       | `min_volume`-`max_volume`
+Type | Interface                     | Value type | Properties               | Description
+-----|-------------------------------|------------|--------------------------|------------------
+in   | cmd.lock.get_report           | null       |                          |
+in   | cmd.lock.set                  | bool       |                          | Use true to secure a lock and false to unsecure
+in   | cmd.lock.set_with_code        | str_map    |                          | Used to lock/unlock locks required PIN/RFID, {“op”:”lock”, ”code_type”:”pin”, ”12345” }
+out  | evt.lock.report               | bool_map   | timeout_s, lock_type     | value = {"is_secured":true, "door_is_closed":true, "bolt_is_locked":true, "latch_is_closed":true}
+-|||   
+in   | cmd.open.get_report           | null       |                          | 
+out  | evt.open.report               | bool       | true = open              | Used to report if the door is open or closed
```

Comment:
it's completely outside of this pr, but related to locks...

shouldn't we consider deprecating this info in favor of creating "virtual" sensor_contact services mapping this value? It's not only the app's UI that has to deal with this info coming from different sources (here, doorman service, sensor_contact), but it also causes problems in creating automations, and probably other parts of the whole system

Reply:
Probably I will not resolve it here as I don't know what shall I do... Maybe we can leave it and fix sometime later?